README.md

```diff
@@ -18,51 +18,18 @@ Resources requirements have been optimized to minimize failures in our computing
-| Quality Check Standard | v1.0.0          | Perform ~10 different QC analyses on a BAM file and compile the results using [MultiQC](https://multiqc.info/).           | [Specification](https://rfcs.stjude.cloud/branches/rfcs/qc-workflow/0002-quality-check-workflow.html) | [Workflow](./workflows/qc/quality-check-standard.wdl)                                                                          | ![In Production](https://img.shields.io/static/v1?label=Status&message=Production&color=green&style=flat-square)    |
-| ESTIMATE               | v1.0.0 (*beta*) | Runs the [ESTIMATE software package](https://bioinformatics.mdanderson.org/estimate/) on a feature counts file.           | None                                                                                                  | [Workflow](./workflows/rnaseq/ESTIMATE.wdl)                                                                                    | ![In Development](https://img.shields.io/static/v1?label=Status&message=Development&color=orange&style=flat-square) |
-| Calculate Gene Lengths | N/A             | Produces a gene length file from a GTF.                                                                                   | None                                                                                                  | [Workflow](./workflows/rnaseq/calc-gene-lengths.wdl)                                                                           | ![In Production](https://img.shields.io/static/v1?label=Status&message=Production&color=green&style=flat-square)    |
-| Build BWA References   | N/A             | Builds reference files used by the [BWA aligner](https://github.com/lh3/bwa).                                             | None                                                                                                  | [Workflow](./workflows/general/bwa-db-build.wdl)                                                                               | ![In Production](https://img.shields.io/static/v1?label=Status&message=Production&color=green&style=flat-square)    |
-| BAM to FASTQs          | v1.0.0          | Split a BAM file into read groups, then read 1 FASTQs and  read 2 FASTQs.                                                 | None                                                                                                  | [Workflow](./workflows/general/bam-to-fastqs.wdl)                                                                              | ![In Production](https://img.shields.io/static/v1?label=Status&message=Production&color=green&style=flat-square)    |
```

Comment:
Are you planning to reintroduce this section later with up-to-date information? Or is this intended as a permanent deletion?

Reply:
Yes, I intend for this section to come back. IDK if this format is the best format we can present the information in (maybe it is), but we will definitely bring this info back.